### DISCUSSION TITLE
[vidstack] dev mode!

### DISCUSSION THREAD

**Discussion by ghost** [vidstack/player#920] [Q&A]
In documentation i found: 
![image](https://github.com/vidstack/player/assets/32639207/8361d578-4466-44fb-bf51-da327ce99f26)

I am using Nextjs v13.x when building the app, vidstack is still in dev mode. I have NODE_ENV=production in my .env file that my project uses but what variable does vidstack expect it to be a production import code? Or how do I setup this?

**Comment by mihar-22** [comment]
If you're trying to use it inside the `/app` directory then we don't support that in the current release. Coming very soon in the next release.

**Reply by ghost** [reply]
I use it inside `/pages` old syntax
